source/v2/greenlanes-openapi.yaml

```diff
@@ -115,7 +115,13 @@ paths:
     get:
       summary: Returns Category Assessments for the requested Goods Nomenclature
       description: >
-        Returns the requested Goods Nomenclature together with relevant data including a list of the different Category Assessments for the Goods Nomenclature
+        Returns the requested Goods Nomenclature together with relevant data including a list of the different Category Assessments applicable to this Goods Nomenclature<br>
+        <br>
+        Each applicable Category Assessment will provide a primary category together with any potential exemptions. If at least one of the exemptions for that Category Assessment are met then that Category Assessment does not apply.<br>
+        <br>
+        The effective category for the GoodsNomenclature is the most restrictive category from those Category Assessments determined by the API consumers filtering to be applicable (ie not exempted).<br>
+        <br>
+        If no Category Assessments are applicable, or all if applicable Category Assessments are exempted, then the trade falls under a default category assessment of Category 3.
```

Comment:
Should this be possible Category Assessment, instead of applicable Category Assessment